dspec_tester.d

```diff
@@ -39,6 +39,7 @@ int main(string[] args)
     auto helpInformation = getopt(
         args,
         "l|lines", "Show the line numbers on errors", &config.printLineNumbers,
+        "compiler", "D compiler to use", &config.dmdBinPath,
```

Comment:
I suggest you delete `config.dmdBinPath = environment.get("DMD", "dmd");` and stick to command-line only args - it makes things easier to debug when the code is more explicit.